deployment/bootstrap-template.yaml

```diff
@@ -0,0 +1,580 @@
+Description: This stack includes resources needed to deploy AWS CDK apps into this environment
```

Comment:
This already has the removal of ECR removed. I've also already bootstrapped all the HCBS accounts.